todoapp/users/permissions.py

```
@@ -0,0 +1,7 @@
+from rest_framework import permissions
+
+
+class IsOwner(permissions.BasePermission):
+
+    def has_object_permission(self, request, view, obj):
```

Ruff (0.12.2)

Missing return type annotation for public function `has_object_permission`

(ANN201)